src/services/external/excalidraw-backend/excalidraw.server.ts

```diff
@@ -243,6 +248,9 @@ export class ExcalidrawServer {
             serverBroadcastEventHandler(roomID, data, socket);
             this.resetCollaboratorModeTimer(socket);
           });
+          socket.on(SCENE_INIT, (roomID: string, data: ArrayBuffer) => {
+            socket.broadcast.to(roomID).emit(CLIENT_BROADCAST, data);
```

Comment:
didn't we agree that the event must be targeting a socket, rather than the whole room?